src/components/overview/ImageGallery.jsx

```diff
@@ -5,7 +5,7 @@ import json from './styleSample.json';
 
 const ImageGallery = (props) => {
 
-  var photos = json.results[0].photos;
+  var photos = props.styles[props.styleIndex].photos;
```

Comment:
Not a big deal or something that needs to change but something you can do to make the code a little bit more readable is decompose props. 
For instance: `const ImageGallery = ({ styles, styleIndex}) => {....`
And then on Line 8 you could do this: `var photos = styles[stylesIndex].photos`